exercises/affine-cipher/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "affine-cipher",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
This should probably be 2.0.0 instead.

From https://github.com/exercism/problem-specifications#major-version-changes:
```Markdown
MAJOR version changes

The MAJOR version should be changed when the test suite is modified in a fundamentally incompatible way.

There are examples of changes requiring a MAJOR version change:

    A new "property" (test type).
    Renaming a "property".
    Insertion, deletion or renaming of keys in the test data object.
    Changing the type of one of the test data keys.

MAJOR changes should be expected to break even well-behaved test generators.
```

Reply:
Okay, thank you.
I wasn't sure which one it should be, but you are very right.  It would break any generator's logic.